ups/verify_ap_hits2015.table

```diff
@@ -0,0 +1 @@
+setupRequired(obs_decam)
```

Comment:
Likewise my knowledge of EUPS but this package is modeled off of [validation_data_cfht](https://github.com/lsst/validation_data_cfht), which has no `.cfg` file.